service/src/main/kotlin/fi/espoo/evaka/assistanceneed/decision/AssistanceNeedDecisionCitizenController.kt

```diff
@@ -24,8 +24,9 @@ class AssistanceNeedDecisionCitizenController(
     private val accessControl: AccessControl,
     private val assistanceNeedDecisionService: AssistanceNeedDecisionService
 ) {
+    // TODO: remove after 7.9.2022
```

Comment:
Today is 8.9.2022 🤔

Reply:
I guess I was a bit optimistic about the schedule 😄